internal/jujuapi/modelmanager_e2e_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 Canonical.
+
+package jujuapi_test
+
+import (
+	"time"
+
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/juju/juju/api/base"
+	"github.com/juju/juju/api/client/modelmanager"
+	"github.com/juju/juju/core/life"
+	"github.com/juju/juju/core/status"
+	"github.com/juju/juju/state"
+	"github.com/juju/version/v2"
+	gc "gopkg.in/check.v1"
+
+	"github.com/canonical/jimm/v3/internal/testutils/jimmtest"
+)
+
+type modelE2eManagerSuite struct {
+	websocketE2ESuite
+}
+
+var _ = gc.Suite(&modelE2eManagerSuite{})
+
+func (s *modelE2eManagerSuite) TestListModelSummaries(c *gc.C) {
```

Comment:
What do you think about moving this test outside of `internal/jujuapi`. The spec states we'll write these new tests in `testing/e2e/`, a nice step towards integration tests that don't need to reach for internals.